pytket_qir/gatesets/base.py

```diff
@@ -38,13 +40,18 @@ class OpName(Enum):
     AND = "and"
     OR = "or"
     XOR = "xor"
+    INT = "integer"
+    BOOL = "bool"
+    RES = "result"
```

Comment:
These look like types rather than ops?

Reply:
That's right. Maybe I should rename these datatypes something like:: `FuncName`, `FuncSpec` etc ?

Reply:
Done.